.drone.star

```diff
@@ -270,11 +270,89 @@ config = {
 			'numberOfParts': 8,
 			'cron': 'nightly'
 		},
+		'api-scality-artesca-remote-smoke': {
```

Comment:
maybe this can be put into a function to share the code with `api-scality8-remote-smoke` ?

Reply:
done - the code added to `acceptance()` will work in any of the apps (won't really be used, but won't break anything) So that can be copied around the apps next time that happens.